<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab14</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="addAButton">
        <property name="text">
         <string>Add a task</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteAButton">
        <property name="text">
         <string>Delete a task</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showAssocAButton">
        <property name="text">
         <string>Show categories of a task</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="addLinkButton">
        <property name="text">
         <string>Add a link</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteLinkButton">
        <property name="text">
         <string>Delete a link</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addBButton">
        <property name="text">
         <string>Add a category</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteBButton">
        <property name="text">
         <string>Delete a category</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showAssocBButton">
        <property name="text">
         <string>Show all tasks of a category</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="aTable">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>120</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>120</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>120</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>204</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Task</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time of the task</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Place of the task</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTableWidget" name="linkTable">
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>306</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Task</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Category</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QTableWidget" name="bTable">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>120</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>120</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>120</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>625</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Category</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="importLinksButton">
        <property name="text">
         <string>Import all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportButton">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
